README.md

```diff
@@ -16,7 +16,9 @@ Having a list of our presentations on GitHub has the following benefits:
 
 | Date       | Event                                                                                                                               | Presentation Title                                               | Topic(s)                                                                      | Video/Slides                                                                                                                                                                                          |
 |:-----------|:------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------|:------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2019-09-24 | [DevTest Conference North Leeds](https://www.softwaretestingnews.co.uk/products/devtest-conference-north/programme/)                | Adapt and Adopt a tester: (r)evolutionary story                  | Quality Assurance                                                             | [Video](https://www.mirobarsa.com/wp-content/uploads/2019/09/National-Software-Testing-Conference-North-Leeds%202019-small.mp4) - [Slides](https://www.slideshare.net/secret/5nnBuS2e3AMHDq) |
 | 2019-09-23 | [London Chaos Engineering Community meetup](https://www.meetup.com/London-Chaos-Engineering-Community/events/264887686/)            | Resilience and Chaos Engineering at Hotels.com                   | Resilience Engineering, Chaos Engineering                                     | [Video](https://skillsmatter.com/skillscasts/14534-resilience-and-chaos-engineering-at-hotels-com#video) - [Slides](https://www.slideshare.net/NikolaosKatirtzis/resilience-engineering-at-hotelscom) |
+| 2019-09-19 | [Codemotion Webinar](https://events.codemotion.com/webinars/when-they-really-understood-to-be-a-team/)                              | When they really understood to be a team                         | Quality Assurance, team management                                            | [Video](https://events.codemotion.com/webinars/when-they-really-understood-to-be-a-team/) - [Slides](https://www.slideshare.net/secret/CYgdjqNqpg1hsn) |
```

Comment:
Nice, thanks for the PR!
Is there a video for this talk? The link you provided doesn't include one.

Reply:
Hi Nikos!
Codemotion org needs some time to upload a video, but it will be here when ready (in this page u can see "soon available"). If u prefer, I can remove it and add it again when the video is available.... let me know :-)

Reply:
Cool, in that case let's leave it as it is.
Thanks for the PR :)